po/R-ca.po

```diff
@@ -0,0 +1,293 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: pkgdown 2.0.6.9000\n"
+"POT-Creation-Date: 2022-11-25 22:59+0100\n"
+"PO-Revision-Date: 2023-08-16 00:23+0200\n"
+"Last-Translator: jmaspons <joanmaspons@gmail.com>\n"
+"Language: ca\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Lokalize 23.04.3\n"
+
+#: build-404.R:11
+msgid "Page not found (404)"
+msgstr "No s'ha trobat la pàgina (404)"
+
+#: build-404.R:12
+msgid "Content not found. Please use links in the navbar."
+msgstr ""
+"No s'ha trobat el contingut. Useu els enllaços de la barra de navegació."
+
+#: build-articles.R:381 navbar.R:136 navbar.R:143 navbar.R:158
+msgid "Articles"
+msgstr "Articles"
+
+#: build-articles.R:430
+msgid "All vignettes"
+msgstr "Totes les vinyetes"
+
+#: build-footer.R:23
+msgid "Developed by"
+msgstr "Desenvolupat per"
+
+#: build-footer.R:28
+msgid "Site built with <a href=\"%s\">pkgdown</a> %s."
+msgstr "Lloc construït amb <a href=\"%s\">pkgdown</a> %s."
+
+#: build-home-authors.R:69
+msgid "More about authors..."
+msgstr "Més sobre els autors..."
+
+#: build-home-authors.R:72
+msgid "Developers"
+msgstr "Desenvolupadors"
+
+#: build-home-authors.R:77 render.R:96
+msgid "Authors"
+msgstr "Autors"
+
+#: build-home-authors.R:169
+msgid "author"
+msgstr "autor"
+
+#: build-home-authors.R:170
+msgid "compiler"
+msgstr "compilador"
+
+#: build-home-authors.R:171
+msgid "contractor"
+msgstr "contractista"
+
+#: build-home-authors.R:172
+msgid "contributor"
+msgstr "col·laborador"
+
+#: build-home-authors.R:173
+msgid "copyright holder"
+msgstr "titular dels drets d'autor"
+
+#: build-home-authors.R:174
+msgid "maintainer"
+msgstr "mantenidor"
+
+#: build-home-authors.R:175
+msgid "data contributor"
+msgstr "col·laborador de dades"
+
+#: build-home-authors.R:176
+msgid "funder"
+msgstr "patrocinador"
+
+#: build-home-authors.R:177
+msgid "reviewer"
+msgstr "revisor"
+
+#: build-home-authors.R:178
+msgid "thesis advisor"
+msgstr "supervisor de tesi"
+
+#: build-home-authors.R:179
+msgid "translator"
+msgstr "traductor"
+
+#: build-home-citation.R:36
+msgid "Citing %s"
+msgstr "Com citar %s"
+
+#: build-home-citation.R:91
+msgid "Authors and Citation"
+msgstr "Autors i citació"
```

Comment:
I would use "Autors i cita" or even "Autors i referències"

Reply:
«cita» doesn't seem to have the proper meaning acoording to ([dlc](https://dlc.iec.cat/Results?DecEntradaText=cita&AllInfoMorf=False&OperEntrada=0&OperDef=0&OperEx=0&OperSubEntrada=0&OperAreaTematica=0&InfoMorfType=0&OperCatGram=False&AccentSen=False&CurrentPage=0&refineSearch=0&Actualitzacions=False)). «referències» could work as well as «citació» but should be in singular as it refers to the documented package only. In this case, I would leave the PR's original text.